server/join/join.go

```diff
@@ -214,7 +214,7 @@ func PrepareJoinCluster(cfg *config.Config) error {
 func isDataExist(d string) bool {
 	dir, err := os.Open(d)
 	if err != nil {
-		log.Error("failed to open directory", errs.ZapError(errs.ErrOSOpen, err))
+		log.Info("failed to open directory, maybe first start", errs.ZapError(errs.ErrOSOpen, err))
```

Comment:
```suggestion
		log.Info("failed to open directory, maybe start for the first time", zap.Error(err))
```
Not an Error log, so no need to use `errs`